src/WellIO.jl

```diff
@@ -1,5 +1,47 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
+using Tables
 
+"""
+    load(file)
+Load log well file from `filepath` and load it in defined structure.
+"""
+function load(filepath::String)
+    file = FileIO.query(filepath)
+    data = open(file) do f
+            read(f, String)
+        end
+    #Splitting the data into table and rest of information
+    datatable_block = match(r"~A(.*?)\Z"s, data)
```

Comment:
Bad variable names. You should probably use something like `header` and `table` to refer to the two parts. Right now you are using `datatable_block` for the `table`? Is that the case? See how hard it is to review the code with these names?